src/app/story/directives/podcast.component.ts

```diff
@@ -92,4 +96,7 @@ export class PodcastComponent implements OnDestroy {
     }
   }
 
+  toggleAlternateSummary() {
```

Comment:
getting `Cannot read property 'getContent' of undefined` for a podcast episode without a description set when I click the toggle checkbox 
![image](https://cloud.githubusercontent.com/assets/16007382/24928339/1c6fa790-1ed0-11e7-924f-4be76f88e3a4.png)